runtime/bcutil/ConstantPoolMap.cpp

```diff
@@ -439,23 +327,34 @@ ConstantPoolMap::findVarHandleMethodRefs()
 
 	for (U_16 i = 1; i < _romConstantPoolCount; i++) {
 		if ((J9CPTYPE_INSTANCE_METHOD == _romConstantPoolTypes[i])
-		|| (J9CPTYPE_INTERFACE_INSTANCE_METHOD == _romConstantPoolTypes[i])
+			|| (J9CPTYPE_INTERFACE_INSTANCE_METHOD == _romConstantPoolTypes[i])
 		) {
 			U_16 cfrCPIndex = _romConstantPoolEntries[i];
-			U_32 slot1 = getCPSlot1(cfrCPIndex);
-			U_32 slot2 = getCPSlot2(cfrCPIndex);
-
-			if (isVarHandleMethod(slot1, slot2)) {
-				if (NULL == varHandleMethodTable) {
-					/* Allocate a temporary array for storing indices of VarHandle methodrefs */
-					varHandleMethodTable = (U_16*) j9mem_allocate_memory(_romConstantPoolCount * sizeof(U_16), OMRMEM_CATEGORY_VM);
-					if (NULL == varHandleMethodTable) {
-						_buildResult = OutOfMemory;
-						break;
+			U_32 classIndex = getCPSlot1(cfrCPIndex);
+			U_32 nasIndex = getCPSlot2(cfrCPIndex);
```

Comment:
This can be moved into the `if ((sizeof(VARHANDLE_CLASS_NAME) - 1) == classNameLength) {` block - always aim for the shortest lifetime for variables as possible as it makes reasoning about the code simpler